lib/widgets/action_sidebar.dart

```diff
@@ -0,0 +1,219 @@
+import 'package:flutter/cupertino.dart';
```

Comment:
I wonder if it would make more sense to move this (and maybe the scaffold?) into the navigation directory. WDYT?

Reply:
I'm not sure if it do really belongs to navigation since it just replaces app bar and must be on same level as scaffold. About moving scaffold, I think another idea is to group widgets that are responsible for layout.

Reply:
Changes:
https://github.com/student-hub/acs-upb-mobile/pull/250/commits/31536835b8d109bd3ef7b078038b0eafbd696c57#diff-91af2d9745e86244940959ec39c99cca9b8b43c1502042d85f16fcc6a311030bR179-R180
https://github.com/student-hub/acs-upb-mobile/pull/250/commits/31536835b8d109bd3ef7b078038b0eafbd696c57#diff-91af2d9745e86244940959ec39c99cca9b8b43c1502042d85f16fcc6a311030bR179-R180

Reply:
Well, the bottom navigation bar *is* in the navigation directory, and the side bar is  similar I guess.
It's a bit hard to define what should go into "layout" I think, as essentially everything is a layout...

Reply:
I agree that scaffold must be placed in same directory as bottom navigation bar but not sure if navigation folder is the best place because there is already  routes.dart and as I do remember @RazvanRotaru  already added some classes that manages navigation.

Reply:
Navigation changes:
https://github.com/student-hub/acs-upb-mobile/tree/web/layout/navigation/lib/navigation

Reply:
I agree with Ioana, this widget is welcomed in /navigation/view